src/mem_usage.txt

```diff
@@ -0,0 +1,40 @@
+Release/RF24Log.elf:     file format elf32-avr
+00000252 l     F .text	00000006 _ZN14RF24LogHandler11setLogLevelEh
+000004c8 l     F .text	00000022 _ZN14RF24LogHandler3logEhPK19__FlashStringHelperS2_PPv
+000004ea l     F .text	00000022 _ZN14RF24LogHandler3logEhPK19__FlashStringHelperPKcPPv
+0000050c l     F .text	00000050 _ZN10RF24Logger4infoIPK19__FlashStringHelperEEvS3_T_z.constprop.15
+00800451 l     O .bss	00000002 rf24Logger
+0000055c l     F .text	00000050 _ZN10RF24Logger5errorIPKcEEvPK19__FlashStringHelperT_z.constprop.14
+000005ac l     F .text	00000050 _ZN10RF24Logger4warnIPKcEEvPK19__FlashStringHelperT_z.constprop.13
+000005fc l     F .text	00000050 _ZN10RF24Logger4infoIPKcEEvPK19__FlashStringHelperT_z.constprop.12
+0000064c l     F .text	00000050 _ZN10RF24Logger5debugIPKcEEvPK19__FlashStringHelperT_z.constprop.11
+0000069c l     F .text	00000050 _ZN10RF24Logger5traceIPKcEEvPK19__FlashStringHelperT_z.constprop.10
+000006ec l     F .text	00000050 _ZN10RF24Logger5errorIPK19__FlashStringHelperEEvS3_T_z.constprop.9
+0000073c l     F .text	00000050 _ZN10RF24Logger4warnIPK19__FlashStringHelperEEvS3_T_z.constprop.8
+0000078c l     F .text	00000050 _ZN10RF24Logger5debugIPK19__FlashStringHelperEEvS3_T_z.constprop.7
+000007dc l     F .text	00000050 _ZN10RF24Logger5traceIPK19__FlashStringHelperEEvS3_T_z.constprop.6
+0000082c l     F .text	00000050 _ZN10RF24Logger3logIPKcEEvhPK19__FlashStringHelperT_z.constprop.5
+0000087c l     F .text	00000050 _ZN10RF24Logger3logIPK19__FlashStringHelperEEvhS3_T_z.constprop.4
+000009c2 l     F .text	000000ac _ZN20RF24StreamLogHandler14appendLogLevelEh
+00800112 l     O .data	00000006 _ZL16rf24logLevelWarn
+00800100 l     O .data	00000006 _ZL17rf24logLevelTrace
+00800118 l     O .data	00000006 _ZL17rf24logLevelError
+0080010c l     O .data	00000006 _ZL16rf24logLevelInfo
+00800106 l     O .data	00000006 _ZL17rf24logLevelDebug
+00000a6e l     F .text	00000268 _ZN20RF24StreamLogHandler12appendFormatEcPPv
+00000cd6 l     F .text	00000090 _ZN20RF24StreamLogHandler15appendTimestampEv
+00000d66 l     F .text	0000009e _ZN20RF24StreamLogHandler5writeEhPK19__FlashStringHelperS2_PPv
+00000e04 l     F .text	0000008e _ZN20RF24StreamLogHandler5writeEhPK19__FlashStringHelperPKcPPv
+00000fd6 l     F .text	00000080 _GLOBAL__I_65535_0_RF24DualLogHandler.cpp.o.1933
+0080044c l     O .bss	00000005 rf24SerialLogHandler
+00800130 l     O .data	0000000e _ZTV20RF24StreamLogHandler
+
+Core:
+.bss  0x002
+.data 0x000
+.text 0x40A
```

Comment:
We have here a bit bigger FLASH usage, probably because of those methods being overriden.